forms/admin/admindashboards/config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form>
```

Comment:
Suggest adding `basei18nUri="admindashboards:configform."` to the `form` element. Then can use convention for all the lablling/icons rather than putting the full i18n on the field `label`, for example.

Reply:
Probably better not; as this is a form fragment that gets merged into all the other widget config forms...